src/components/Header/Header.js

```diff
@@ -5,20 +5,18 @@ import { Link } from 'react-router-dom';
 const Header = () => {
 
   const toggleLink = () => {
-    const homeLink = document.querySelectorAll('.home-link')
-    const profileLink = document.querySelectorAll('.profile-link')
-    homeLink.forEach(e => e.classList.toggle('hidden'))
-    profileLink.forEach(e => e.classList.toggle('hidden'))
+    const homeLink = document.querySelector('.home-link')
+    const profileLink = document.querySelector('.profile-link')
+    homeLink.classList.toggle('hidden')
+    profileLink.classList.toggle('hidden')
   }
 
   return (
     <header className='header'>
       <h1 className='veg-share-title'>Garden Party</h1>
       <div className='header-links'>
-        <button className='home-link link-button hidden'>
+        <button className='link-button'>
```

Comment:
This was my code, but if there's time we may want to improve this - button only works right now if you click on the text (the Link). Tried to just widen the Link, but wasn't cooperating.

Reply:
Hmm, that is weird. We can definitely look into fixing that later today or even tomorrow.

Reply:
Yeah that's weird, I wonder if wrapping in a link would help?